src/main/resources/import.sql

```diff
@@ -2,3 +2,9 @@
 insert into `people` (name, current_hp, hp_max, current_mp, mp_max) values ('Loup Ardent', '12', '12', '6', '6');
 insert into `people` (name, current_hp, hp_max, current_mp, mp_max) values ('Hardy Xav', '1000', '1000', '1000', '1000');
 insert into `people` (name, current_hp, hp_max, current_mp, mp_max) values ('Conan le Barbare', '14', '18', '4', '4');
+
+
+/* Item */
+insert into `item` (name, price, slot, people_id) values ('Dent de dragon', '100', '1', '1');
+insert into `item` (name, price, slot, people_id) values ('salicorne', '2', '1', '2');
+insert into `item` (name, price, slot, people_id) values ('gousse de mandragore', '5', '1', '3');
```

Comment:
Use integers (no quote `'`) when possible.

Reply:
agreed and modified.